packages/kit/src/exports/vite/index.js

```diff
@@ -132,6 +132,23 @@ export async function sveltekit() {
 			hydratable: true,
 			...svelte_config.compilerOptions
 		},
+		experimental: {
+			dynamicCompileOptions: ({ filename, code, compileOptions }) => {
+				const options =
+					svelte_config.vitePlugin?.experimental?.dynamicCompileOptions?.({
+						filename,
+						code,
+						compileOptions
+					}) ?? {};
+
+				const basename = path.basename(filename);
+				if (basename.startsWith('+layout') || basename.startsWith('+page')) {
+					options.accessors = true;
+				}
+
+				return options;
+			}
+		},
```

Comment:
Is enabling accessors for layouts and pages fewer bytes than adding a `get_snapshot()` (or whatever) method to them that just returns that prop?

Reply:
oh hey i totally forgot that `export const` causes the accessor to be created automatically. it doesn't have to be a function — `export const snapshot = {...}` works fine. this is preferable i think from a typing perspective (which reminds me of an additional TODO)

Reply:
Gosh. I also had completely forgotten about that. Much nicer!